packages/priority-queue/README.md

```diff
@@ -40,7 +40,7 @@ queue.add( ctx2, () => console.log( 'This will be printed second' ) );
 
 _Returns_
 
--   `Object`: Queue object with `add` and `flush` methods.
+-   `WPPriorityQueue`: Queue object with `add` and `flush` methods.
```

Comment:
Aside: Enhancements resulting from #15186 could be an important selling-point in convincing developers why documenting custom types could be beneficial for improved, automated documentation.